crowd_anki/config/config_settings.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
+from enum import Enum
+
+from aqt import mw
+from ..utils.constants import USER_FILES_PATH
+
+ConfigEntry = namedtuple("ConfigEntry", ["config_name", "default_value"])
+
+
+class NoteSortingMethods(Enum):
+    FIELD1 = "field1"
+    FIELD2 = "field2"
+    NO_SORTING = "none"
+    GUID = "guid"
+    FLAG = "flag"
+    TAG = "tag"
+    NOTE_MODEL = "notemodel"
```

Comment:
also looking at the actual sort method - this should be NOTE_MODEL_NAME

Reply:
underscores!

Reply:
Consider_it_done!

Reply:
🎉